src/document-creator/document-creator.mdx

```diff
@@ -30,41 +32,94 @@ Before you construct your configuration file(`.json` file), please ensure you ha
 - [Token Registry (for transferable document)](./token-registry)
 - [Configured DNS](/dns-txt#how-to-create-dns-txt-record)
 - [Custom Renderer](./custom-template)
+- [Document Storage](/infra-templates#storage)
 
 Please refer to the approprate links to prepare them.
 
+---
+
 ## Configuration file
 
 The configuration file is a JSON file that contains information to configure the application to suit your business needs.
 
 ### Config file structure
 
+Below is an example of the configuration file.
+
 ```json
 {
   "network": "ropsten",
   "wallet": "{\"address\":\"6a36c563a5350d7be66c801f901a67...\", ...}}",
-  "forms": [{...}]
+  "forms": [{...}],
+  "documentStorage": {
+    "apiKey": "kNb15YYZ6N1zBlYd25cjj8PLgK6YAuvN9Gf7fPM1",
+    "url": "https://api-ropsten.tradetrust.io/storage"
+  }
 }
 ```
 
-### `"network"` field
+Let's break down the required fields for this config file.
+
+We will need:
+
+- [Network field](#network-field)
+- [Wallet field](#wallet-field)
+- [Forms field](#forms-field)
+- [Document Storage field](#document-storage-field)
+
+---
+
+## Network field
+
+The `"network"` field is a string that refers to the network in which the document is created in.
+
+As of now, we only cater to 3 networks.
+
+- Mainnet : `"homestead"`
+- Ropsten testnet : `"ropsten"`
+- Rinkeby testnet : `"rinkeby"`
+
+---
+
+## Wallet field
+
+The `"wallet"` field is a string that refers to your ethereum wallet.
 
-The `network` field is a string that refers to the network in which the document is created in.
+If you have the wallet you can convert the wallet into a string by following these steps:
 
-### `"wallet"` field
+1. open your browser. (preferably mozella firefox)
 
-The `wallet` field is a string that refers to your etherum wallet.
+2. open inspect element on any page.
 
-To convert the wallet into a string like the above example, please use this [online tool](https://tools.knowledgewalls.com/jsontostring).
+3. go to the console tab.
 
-### Forms structure
+4. type: "Node", and press enter.
 
-The `forms` field is an array of form object which expect `name`, `type`, `defaults` and `schema`. `attachments` can also be added.
+5. type: "JSON.stringify({wallet})".
```

Comment:
can just say

If you already have an existing wallet.json, the only thing you need to do is to run JSON.stringify over the contents in order to serialise it into a string.

This can be achieved using either the NodeJS console or the browser console, using the command `JSON.stringify({wallet})` where `{wallet}` is the contents of your wallet.json file.

For example: 

> w = { "address ": "6a36c563a5350d7be66c801f901a67 ... }
> JSON.stringify(w)

Reply:
ok